### PR TITLE
fix keybinds document

### PR DESCRIPTION
**Pull request by hitsuji-no-shippo**
#693

### CODE REVIEW

File: pkg/gui/keybindings.go [hunk 1]
```diff
@@ -333,6 +333,13 @@ func (gui *Gui) GetInitialKeybindings() []*Binding {
 			Handler:     gui.prevScreenMode,
 			Description: gui.Tr.SLocalize("prevScreenMode"),
 		},
+		{
+			ViewName:    "",
```
Comment (jesseduffield) [inline pkg/gui/keybindings.go]: given this actually isn't a global keybinding, we don't want to add this keybinding here. We have two places where we set this keybinding: one here:
```
if listView.viewName != "commits" {
			bindings = append(bindings, &Binding{ViewName: listView.viewName, Contexts: []string{listView.context}, Key: gui.getKey("universal.startSearch"), Modifier: gocui.ModNone, Handler: gui.handleOpenSearch})
		}
```
and one here:
```
		{
			ViewName: "commits",
			Key:      gui.getKey("universal.startSearch"),
			Modifier: gocui.ModNone,
			Handler:  gui.handleOpenSearchForCommitsPanel,
		},
```
So I would add the descriptions there :)

Comment (hitsuji-no-shippo) [reply pkg/gui/keybindings.go]: Hi @jesseduffield .

Sorry. I don't understand what to do next. I think English learners are the reason I don't understand.

Would you tell me what to do next?

Comment (jesseduffield) [reply pkg/gui/keybindings.go]: All good :)

In line `pkg/gui/keybindings.go:1457` we can do this:
```
			bindings = append(bindings, &Binding{ViewName: listView.viewName, Contexts: []string{listView.context}, Key: gui.getKey("universal.startSearch"), Modifier: gocui.ModNone, Handler: gui.handleOpenSearch, Description: gui.Tr.SLocalize("startSearch")})
```
(not we're adding `Description: gui.Tr.SLocalize("startSearch")` to this line).

Then in `pkg/i18n/english.go` at the bottom of the file we can add:
```
		}, &i18n.Message{
			ID:    "startSearch",
			Other: "start search",
```

Then we can run `go run scripts/generate_cheatsheet.go` to generate the new keybindings cheatsheet. Does that make sense?

Comment (hitsuji-no-shippo) [reply pkg/gui/keybindings.go]: Thank you for tell me. Sorry for my late reply.
I think this response comment different from what you expect.

I made the following changes as you told me:

#### Add `Description: gui.Tr.SLocalize("startSearch")`

```golang
bindings = append(bindings, &Binding{ViewName: listView.viewName, Contexts: []string{listView.context}, Key: gui.getKey("universal.startSearch"), Modifier: gocui.ModNone, Handler: gui.handleOpenSearch, Description: gui.Tr.SLocalize("startSearch")})
```

```golang
{
	ViewName: "commits",
	Key:      gui.getKey("universal.startSearch"),
	Modifier: gocui.ModNone,
	Handler:  gui.handleOpenSearchForCommitsPanel,
	Description: gui.Tr.SLocalize("startSearch"),
},
```

#### Delete universal.startSearch

Delete code

```golang
{
	ViewName:    "",
	Key:         gui.getKey("universal.startSearch"),
	Modifier:    gocui.ModNone,
	Handler:     gui.handleOpenSearch,
	Description: gui.Tr.SLocalize("startSearch"),
},
```

I also think that fine with this than change of my pull request.

I'm concerns about the Menu Panel in the generated docs, so add a description key.

#### Places of concern in generated docs

```Markdown
## Menu Panel

<pre>
  <kbd>/</kbd>: start search
</pre>
```

#### Code to add `description` key

https://github.com/jesseduffield/lazygit/blob/7df4b736cf1b513358ac1ed9609bc6ec644cb575/pkg/gui/keybindings.go#L894-L905

Update this pull request until 12:00 on March 13 (UTC).

Thank you.

**Comment by jesseduffield** [comment]
LGTM! thanks for making this

**Pull request merged**